src/otx/algo/detection/heads/base_sampler.py

```diff
@@ -103,53 +104,6 @@ def sample(
         Returns:
             :obj:`SamplingResult`: Sampling result.
         """
-        gt_bboxes = gt_instances.bboxes  # type: ignore[attr-defined]
```

Comment:
Is this code unnecessary now?

Reply:
OTX use random sampler or pseudo sampler, and they have their own sample function. So we don't need this